src/components/AboutModal/__mocks__/mockAboutModal.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { Button } from '../../Button';
+import { AboutModal } from '../index';
+import logo from 'patternfly/dist/img/logo-alt.svg';
+
+export class MockAboutModal extends React.Component {
+  constructor() {
+    super();
+    this.state = { showModal: false };
+    this.open = this.open.bind(this);
+    this.close = this.close.bind(this);
+  }
+  open() {
+    this.setState({ showModal: true });
+  }
+  close() {
+    this.setState({ showModal: false });
+  }
+  render() {
+    return (
+      <div>
+        <Button bsStyle="primary" bsSize="large" onClick={this.open}>
+          Launch about modal
```

Comment:
@jeff-phillips-18 total nit, but can the launch button label use initial caps, making it "Launch About Modal" ?